api/migrations/20200626194403-removeOrphanedDocs.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+var ObjectID = require('mongodb').ObjectID;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.connection.connect(db.connectionString, { native_parser: true })
+    .then(async (mClient) => {
+      var nrptiCollection = mClient.collection('nrpti');
+      console.log('Getting all records');
+      const records = await nrptiCollection.find(
+        {
+          _schemaName: {
+            $in: [
+              'AdministrativePenalty',
+              'AdministrativeSanction',
+              'Agreement',
+              'Certificate',
+              'ConstructionPlan',
+              'CourtConviction',
+              'Inspection',
+              'ManagementPlan',
+              'Order',
+              'Permit',
+              'RestorativeJustice',
+              'SelfReport',
+              'Ticket',
+              'Warning'
+            ]
+          }
+        }
+      ).toArray();
+      var recordDocIds = [];
+      for (let i = 0; i < records.length; i++) {
+        if (records[i].documents && records[i].documents.length > 0) {
+          records[i].documents.forEach(id => {
+            recordDocIds.push(ObjectID(id).toString());
+          });
```

Comment:
Careful here once converting to await

Reply:
3fb0c0014b44f40a4f53dd1c437c15124582029f